.gitignore

```diff
@@ -11,3 +11,4 @@ apps/worker/doc/edoc.css
 doc/users-guide/overview-eps-converted-to.pdf
 integration-coverage/
 .rebar
+rebar.lock
```

Comment:
I *think* I would like to have this checked in. If it works similar to the `Gemfile.lock` for the ruby bundler, that is.